apps/dashboard/src/app/(app)/team/[team_slug]/(team)/~/settings/general/TeamGeneralSettingsPageUI.tsx

```diff
@@ -57,6 +58,13 @@ export function TeamGeneralSettingsPageUI(props: {
         initialVerification={props.initialVerification}
         isOwnerAccount={props.isOwnerAccount}
       />
+      <TeamDedicatedSupportCard
+        teamId={props.team.id}
+        teamSlug={props.team.slug}
+        billingPlan={props.team.billingPlan}
+        channelType={props.team.dedicatedSupportChannel?.type}
```

Comment:
Getting this error on the type after I added the type to TeamResponse. Not sure why?

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/vKjRiOG6EUjD7XH4cppU/066de608-bc9e-46b0-b428-078d1d898de2.png)